src/common.jl

```diff
@@ -80,6 +74,33 @@ function solve{uType,tType,isinplace,algType<:ODEJLAlgorithm,F}(prob::AbstractOD
                           minstep=dtmin,
                           initstep=dt,
                           points=points)
+    elseif typeof(alg) <: ode4
+        ts,timeseries_tmp = ODE.ode4(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
+    elseif typeof(alg) <: ode4ms
+        ts,timeseries_tmp = ODE.ode4ms(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
+    elseif typeof(alg) <: ode4s
+        ts,timeseries_tmp = ODE.ode4s(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
     end
```

Comment:
Oh I thought you meant type constructor. My bad for the confusion. No, the type declaration ode_alg::T with `T<ODEJLAlgorithm` is necessary for dispatch to work correctly, otherwise it doesn't know which `solve` function to use.

And sure, I'll change the others to use `kwargs`.